.github/workflows/shared-tests-pytests.yml

```
@@ -69,7 +69,7 @@ jobs:
       PYTHON: ${{ matrix.python-version }}
 
     steps:
-      - uses: actions/checkout@v6
+      - uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6.0.2
         with:
           fetch-depth: ${{ inputs.fetch-depth }}
```

actionlint (1.7.12)

[error] property "fetch-depth" is not defined in object type {allow-prereleases: bool; cache: bool; dependency-manager: string; env: string; fail_on_codecov_error: bool; install_editable: bool; os: string; pipenv-install-options: string; pytest_opts: string; python-versions: string}

(expression)